profiles/apparmor.d/usr.share.openqa.script.openqa

```diff
@@ -19,8 +19,7 @@
   /etc/openqa/database.ini r,
   /etc/openqa/openqa.ini r,
   /proc/meminfo r,
-  /tmp/mojo.tmp rw,
-  /tmp/mojo.tmp.* rw,
+  /tmp/** rw,
   /usr/bin/optipng rix,
```

Comment:
/var/lib/openqa/workers is another option - it doesn't have to be a tmp directory actually. I suggest we don't touch anything at all (beside the worker profile) and put an issue in backlog to do that properly